root/relationship/linkattributetype/RelationshipAttributeTypesList.js

```diff
@@ -68,6 +68,10 @@ const AttributeDetails = ({
             </a>
           </>
         )}
+        {' | '}
+        <a href={'/relationship-attribute/' + attribute.gid}>
+          {l('Documentation')}
+        </a>
```

Comment:
I was super confused why I couldn't find this, and it's 'cause it's currently under `$c.user?.is_relationship_editor`. :) (I think it can be shown even if you're not logged in.)

Reply:
That's from a different file which I was just renaming, but I can change that in a separate commit I guess :)